public/javascripts/Admin/src/Admin.js

```diff
@@ -1,495 +1,70 @@
-        maxZoom: 19,
-        minZoom: 9,
-        zoomSnap: 0.5
-    });
-
-    // a grayscale tileLayer for the choropleth
-    L.mapbox.accessToken = 'pk.eyJ1IjoibWlzYXVnc3RhZCIsImEiOiJjajN2dTV2Mm0wMDFsMndvMXJiZWcydDRvIn0.IXE8rQNF--HikYDjccA7Ug';
-    var choropleth = L.mapbox.map('admin-choropleth', "mapbox.light", {
-        maxZoom: 19,
-        minZoom: 9,
-        legendControl: {
-            position: 'bottomleft'
+    var mapLoaded = false;
+    var graphsLoaded = false;
+    var mapData = LayerController();
+    var map;
+    var auditedStreetLayer;
+    var params = {
+        regionColors: [
+            '#08306b', '#08519c', '#08719c', '#2171b5', '#4292c6',
+            '#6baed6', '#82badb', '#9ecae1', '#b3d3e8', '#c6dbef'
+        ],
+        neighborhoodPolygonStyle: {
+            color: '#888',
+            weight: 1,
+            opacity: 0.25,
+            fillColor: "#f00",
+            fillOpacity: 1.0
         },
-        zoomSnap: 0.5
-    });
-    choropleth.scrollWheelZoom.disable();
-
-    // Set the city-specific default zoom and location.
-    $.getJSON('/cityMapParams', function(data) {
-        map.setView([data.city_center.lat, data.city_center.lng]);
-        map.setZoom(data.default_zoom);
-        choropleth.setView([data.city_center.lat, data.city_center.lng]);
-        choropleth.setZoom(data.default_zoom);
-        initializeOverlayPolygon(map, data.city_center.lat, data.city_center.lng);
-    });
-
-    // Initialize the map
-    /**
-     * This function adds a semi-transparent white polygon on top of a map.
-     */
-    function initializeOverlayPolygon(map, lat, lng) {
-        var overlayPolygon = {
-            "type": "FeatureCollection",
-            "features": [{
-                "type": "Feature", "geometry": {
-                    "type": "Polygon", "coordinates": [
-                        [
-                            [lng + 2, lat - 2],
-                            [lng + 2, lat + 2],
-                            [lng - 2, lat + 2],
-                            [lng - 2, lat - 2],
-                            [lng + 2, lat - 2]
-                        ]
-                    ]
-                }
-            }]
-        };
-        var layer = L.geoJson(overlayPolygon);
-        layer.setStyle({color: "#ccc", fillColor: "#ccc"});
-        layer.addTo(map);
-    }
-
-    /**
-     * render points
-     */
-    function initializeNeighborhoodPolygons(map) {
-        var neighborhoodPolygonStyle = {
+        mouseoverStyle: {
+            opacity: 1.0,
+            weight: 3,
+            color: "#000"
+        },
+        mouseoutStyle: {
+           opacity: 0.25,
+           weight: 1 
+        },
+        zoomControl: true,
+        disableScrollWheel: true,
+        mapName: 'admin-choropleth',
+        mapStyle: "mapbox.light",
+        accessToken: 'pk.eyJ1IjoibWlzYXVnc3RhZCIsImEiOiJjajN2dTV2Mm0wMDFsMndvMXJiZWcydDRvIn0.IXE8rQNF--HikYDjccA7Ug'
+    };
+    var mapParams = {
```

Comment:
When I have both `params` and `mapParams`, could I use `choroplethParams` and `mapParams` instead, respectively?

Reply:
I don't think that's great because they are both choropleths and they are both maps. I think they easiest way to differentiate them is by which tab they are in on the admin page. It's a little clunky, but how about `analyticsTabMapParam` and `mapTabMapParams`..? Could also rename the "Map" tab to make this sound better..?